py/static/index.html

```diff
@@ -25,12 +25,15 @@
     <link rel="stylesheet" href={{ static_url("css/react-grid-layout-styles.css") }}>
 
     <!-- Other deps -->
-    <script src={{ static_url("js/react-react.min.js") }}></script>
-    <script src={{ static_url("js/react-dom.min.js") }}></script>
-    <script src={{ static_url("fonts/classnames") }}></script>
+    <script src={{ static_url("js/react.development.js") }}></script>
+    <script src={{ static_url("js/react-dom.development.js") }}></script>
+
     <script src={{ static_url("fonts/layout_bin_packer") }}></script>
-    <script src={{ static_url("js/react-grid-layout.min.js") }}></script>
```

Comment:
Not at all a js expert here, but I think the way I have it configured, it gets compiled into the main.js that is generated. I did it this way because I was having a lot of version conflicts with downloading the js files. yarn should handle the dependencies and compile it automatically.